sw/device/lib/testing/test_coverage.h

```diff
@@ -0,0 +1,16 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Calls the LLVM profiling runtime initialization function.
+ */
+void test_coverage_init_buffer();
+
+/**
+ * Sends the LLVM profile buffer along with its length and CRC32.
+ *
+ * This profile data is raw and must be indexed before it can be used to
+ * generate coverage reports.
+ */
```

Comment:
Do you think you could expand on these comments to describe what they do in a bit more detail (i.e. mostly contractual obligations on callers).